hibike/devices/YogiBear/ui.h

```diff
@@ -0,0 +1,31 @@
+#ifndef UI_H
+#define UI_H
+
+// function prototypes
+void manual_ui();
+
+//A space for constants.
+extern float pwmInput; //Value that is received from hibike and is the goal PWM
```

Comment:
We have removed the UI file as noted in a comment above. All extern variables have been removed and replaced with proper getters/setters